Xamarin.Forms.Platform.Android/StepperRendererManager.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.ComponentModel;
+using Android.Views;
+using AButton = Android.Widget.Button;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public static class StepperRendererManager
+	{
+		public static void CreateStepperButtons<TButton>(IStepperRenderer renderer, out TButton downButton, out TButton upButton)
+			where TButton : AButton
+		{
+			downButton = (TButton)renderer.CreateButton();
+			downButton.Id = Platform.GenerateViewId();
+			downButton.Focusable = true;
+			downButton.Text = "－";
```

Comment:
I have concerns that the default font will not always be the same. May not even contain this symbol. Is it possible in the future to use icons instead of text?

Reply:
Right now, no. We can add the ability to set icons, and that is probably a better option - especially with the new FontIconSource. That is the dream. But, do people actually use this control? :) It is basically 2 buttons with zero customisability. The only reason for this PR is to make the material demos better - setting a page to material, the stepper was the only buttons that weren't material - and there is absolutely no way to customize the font/size/color/anything. I am sure people just create their own - 2 buttons at 36x36, with a 4 space. Boom, you have a stepper, lol.

Reply:
Could we explicitly set the font on these two buttons? Then we wouldn't have to worry about default font changes changing the appearance. (Scaling might still be an issue.)

Reply:
That might be a breaking change as the initial stepper did not have text styles at all.